docs/openapi/schemas.yaml

```diff
@@ -634,3 +637,35 @@ RUMDomainDiscoveryAlert:
     domain:
       description: Domain of the page which triggered the domain discovery alert
       $ref: '#/Domain'
+SlackChannelCreateRequest:
+  type: object
+  required:
+    - imsOrgId
+    - users
+  properties:
+    imsOrgId:
+      description: The ID of the Adobe IMS organization
+      $ref: '#/ImsOrganizationId'
+    users:
```

Comment:
email addresses are not actually users but more hints on existing or to be invited users... call it `emails`?

Reply:
I had initially gone with `emails` but wanted to align it with the Slack client, where the param name is `users` and the type is an array of email addresses: https://github.com/adobe/spacecat-shared/blob/main/packages/spacecat-shared-slack-client/src/clients/elevated-slack-client.js#L407

Will update to `emails` for the API 👍